### ISSUE TITLE
Accessibility issues with designs

### ISSUE THREAD

**Issue by shawnthompson** [canada-ca/design-system-systeme-conception#90]
We have notices that there are accessibility issues on the [Institutional landing page](https://design.canada.ca/coded-layout/institutional_landing_page_guidance.html).

* `ip-cover-img` has information in it specifying the image size "1200 x 726" but the colour contrast of the text fails the required contrast ratio
* page does not pass W3C HTML validation checks
*  In the Contact us and About section, the content if only using heading (`h2` and `h3`) elements in this sections. This is not using proper heading level semantically as per the HTML standards. There should be content in between the heading that are the same level. 

Are you folks testing for accessibility before promoting this? Some departments are going to start using these beta 

CC: @bsouster, @RubyDo, @rlambert27, @JuliannaR

**Comment by RababGomaa** [comment]
Thanks @shawnthompson ! I created separate issue with more description for the failures on 
#91 Accessibility: Multiple occurrences for misuse of `<section>`(Failure WCAG 1.3.1)
#92 Accessibility: Related links that are not grouped in a list (Failure WCAG 1.3.1) 
#93 Accessibility : Failing sufficient contrast due to text on background image (WCAG 1.4.3)
#94 Accessibility : Repetitive and unclear heading text (Failure WCAG 2.4.6)
